packages/analyticsdx-template-lint/src/linter.ts

```diff
@@ -1132,10 +1133,12 @@ export abstract class TemplateLinter<
   private async lintLayout(templateInfo: JsonNode) {
     const { doc, json: layout } = await this.loadTemplateRelPathJson(templateInfo, ['layoutDefinition']);
     if (doc && layout) {
-      await Promise.all([
+      const promises = Promise.all([
         this.lintLayoutCheckVariables(templateInfo, doc, layout),
-        this.lintLayoutCheckNavigationObjects(doc, layout)
+        this.lintLayoutValidationPages(templateInfo, doc, layout)
       ]);
+      this.lintLayoutCheckNavigationObjects(doc, layout);
```

Comment:
This one's not actually async, so we don't really need it in the list of async ones.